DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js

```diff
@@ -1,13 +1,30 @@
-// Copyright 2019 Google LLC
+//Copyright 2020 Google LLC
```

Comment:
nit: Doesn't the google style guide want a space between the "//" and the comment?